app/Http/Controllers/LoginRegisterControllers.php

```diff
@@ -54,4 +59,65 @@ protected function getToken(Request $request)
         return Auth::guard('api')->attempt($request->only('email', 'password'));
     }
 
+    public function forgetPassword()
+    {
+
+        return view("forget-password");
+    }
+
+    public function sendEmail(ForgetPasswordRequest $request)
+    {
+        try {
+
+            $user = User::where('email', $request->email)->first();
+
+            if (!is_null($user)) {
+                $domain = URL::to('/');
+                $url = $domain . '/users/reset-password/' . $user->id;
+                $data = [
+                    "url" => $url,
+                    "email" => $user->email,
+                    "title" => "Reset Password Link",
+                    "body" => "Please click on the below button to reset your password",
+                ];
+
+                Mail::to('k224187@nu.edu.pk')->send(new ForgetPasswordMail($data));
```

Comment:
why there is a hard coded email here?

Reply:
Sir all the emails in DB were fake so for checking purpose I hardcoded this and forget to remove.

Reply:
ok but please take care next time